roles/ceph-rgw/tasks/rgw_create_pools.yml

```diff
@@ -73,12 +73,39 @@
     - item.value.type is not defined or item.value.type == 'replicated'
     - item.value.rule_name | default(ceph_osd_pool_default_crush_rule_name)
 
+- name: get pool details
+  command: "{{ container_exec_cmd }} ceph --connect-timeout=10 --cluster={{ cluster }} --format=json osd pool ls detail"
+  register: result
+  retries: 60
+  delay: 3
+  until: result is succeeded
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  changed_when: false
+  check_mode: false
+
+- name: set rgw_osd_pool_ls_detail fact
+  set_fact:
+    rgw_osd_pool_ls_detail: "{{ result.stdout|from_json }}"
```

Comment:
```suggestion
    rgw_osd_pool_ls_detail: "{{ result.stdout | from_json }}"
```

Reply:
Is `default('{}')` really needed? The task that registers `result` has `check_mode: false`, so `result.stdout` should always be defined.